src/services/outliningElementsCollector.ts

```diff
@@ -86,7 +89,7 @@ function addNodeOutliningSpans(sourceFile: SourceFile, cancellationToken: Cancel
         }
     }
 
-    function visitNonImportNode(n: Node) {

```

Comment:
I feel like the name implies that ignoring imports was intentional, but I can't really see why this might be the case... It can't have been noise given we give spans for _parens_...